static/app/components/replays/videoReplayer.tsx

```diff
@@ -109,13 +133,75 @@ export class VideoReplayer {
     return el;
   }
 
+  /**
+   * Resume timer only if replay is running. This accounts for
+   * playing through "dead air".
+   */
```

Comment:
```suggestion
  /**
   * Resume timer only if replay is running. This accounts for
   * playing through "dead air". This is used only in `setBuffering`. 
   */
```

Reply:
to clarify that in all other places where we "play" a replay, we should be calling `startReplay` instead